limelight/src/test/java/com/team2813/lib2813/limelight/RestLimelightTest.java

```diff
@@ -47,6 +50,20 @@ public void fakeWorks() throws Exception {
 		HttpResponse<String> response = client.send(req, BodyHandlers.ofString());
 		assertEquals("{\"v\":1}", response.body());
 	}
+	
+	@Test
+	public void setFieldMapWorks() throws Exception {
+		String fieldMap;
+		Limelight limelight = createLimelight();
+		try (InputStream is = getClass().getResourceAsStream("frc2025r2.fmap")) {
+      assertNotNull(is);
+			is.mark(Integer.MAX_VALUE);
+      fieldMap = new String(is.readAllBytes(), StandardCharsets.UTF_8);
+			is.reset();
+			limelight.setFieldMap(is, true);
+		}
+		assertEquals(fieldMap, fakeLimelight.getFieldMap());
```

Comment:
I suggest we keep things simple:

```suggestion
		Limelight limelight = createLimelight();
		String resourceName = "frc2025r2.fmap";
		try (InputStream is = getClass().getResourceAsStream(resourceName)) {
			limelight.setFieldMap(is, true);
		}
		String fieldMap = fakeLimelight.getFieldMap();
		assertNotNull(fieldMap);
		
		String expectedFieldMap;
		try (InputStream is = getClass().getResourceAsStream(resourceName)) {
			expectedFieldMap = new String(is.readAllBytes(), StandardCharsets.UTF_8);
		}
		assertEquals(expectedFieldMap, fieldMap);
```